plio/io/isis_serial_number.py

```diff
@@ -108,7 +116,7 @@ class SerialNumberDecoder(pvl.decoder.PVLDecoder):
     serial number. Inherits from the PVLDecoder in planetarypy's pvl module.
     """
 
-    def cast_unquoated_string(self, value):
+    def cast_unquoted_string(self, value):
```

Comment:
Will this break other libraries?

Reply:
I don't think so. `generate_serial_number` is the main usage from this file. We could tick the version # to be safe.